samples/coop_synchronization/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_STDOUT_CONSOLE=y
+# enable to use thread names
+CONFIG_THREAD_NAME=y
+CONFIG_SCHED_CPU_MASK=y
```

Comment:
Worth adding CONFIG_SYS_CLOCK_EXISTS=n to guarantee there are no timer interrupts.

Reply:
I think that's a great idea, when I try the linking fails with an undefined reference to `smp_timer_init()`, which is not that surprising. 
I can't decide which direction I should go from here; filter out `smp` enabled boards OR not introduce `CONFIG_SYS_CLOCK_EXISTS=n`. 
May I ask for your opinion on this?